src/test/java/life/qbic/openbis/openbisclient/TestOpenBisClient.java

```diff
@@ -27,8 +28,8 @@ public static void setUpBeforeClass() {
     config = new Properties();
     config.setProperty(DATASOURCE_URL,
         "https://qbis.qbic.uni-tuebingen.de/openbis/openbis" + IApplicationServerApi.SERVICE_URL);
-    config.setProperty(DATASOURCE_USER, "zxmqw74");
```

Comment:
Luis said that there is an option to remove pw from commits. We should prob find it to be sure

Reply:
done